core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -116,6 +116,17 @@ <h4 class="modal-title">Execution results of Workflow: {{workflowName}}</h4>
               nzSize="default"
             ></nz-avatar>
           </td>
+          <td nzEllipsis>
+            <div style="flex-grow: 2; text-align: center">
```

Comment:
try avoiding styles in .html and add it to the scss file

Reply:
done